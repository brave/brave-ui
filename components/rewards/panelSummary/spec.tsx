/* global jest, expect, describe, it, afterEach */
import * as React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import PanelSummary, { Props } from './index'

const props = {
  grant: {color: '#C12D7C', tokens: 10, converted: 0.25},
  ads: {color: '#C12D7C', tokens: 10, converted: 0.25},
  contribute: {color: '#9752CB', tokens: 10, converted: 0.25},
  donation: {color: '#4C54D2', tokens: 2, converted: 0.25},
  tips: {color: '#4C54D2', tokens: 19, converted: 5.25},
  grants: [
    {
      id: '1',
      tokens: 15,
      converted: 0.75
    },
    {
      id: '2',
      tokens: 10,
      converted: 0.50
    }
  ],
  onActivity: ()=>{}
}

describe('PanelSummary tests', () => {
  const baseComponent = (props: Props) => <PanelSummary id='empty' {...props} />

  describe('basic tests', () => {
    it('matches the snapshot', () => {
      const component = baseComponent(props)
      const tree = create(component).toJSON()
      expect(tree).toMatchSnapshot()
    })

    it('renders the component', () => {
      const wrapper = shallow(baseComponent(props))
      const assertion = wrapper.find('#empty').length
      expect(assertion).toBe(1)
    })
  })
})
