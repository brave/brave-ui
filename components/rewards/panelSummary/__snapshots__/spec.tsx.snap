// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelSummary tests basic tests matches the snapshot 1`] = `
.c0 {
  font-family: Poppins;
  margin-top: 21px;
}

.c1 {
  font-size: 14px;
  font-weight: 600;
  line-height: 1.57;
  -webkit-letter-spacing: 0.4px;
  -moz-letter-spacing: 0.4px;
  -ms-letter-spacing: 0.4px;
  letter-spacing: 0.4px;
  color: #a1a8f2;
  text-transform: uppercase;
}

.c2 {
  font-size: 28px;
  font-weight: 300;
  line-height: 0.79;
  -webkit-letter-spacing: 0.4px;
  -moz-letter-spacing: 0.4px;
  -ms-letter-spacing: 0.4px;
  letter-spacing: 0.4px;
  color: #4c54d2;
  margin-bottom: 26px;
  text-transform: uppercase;
}

.c12 {
  font-size: 22px;
  font-weight: 300;
  line-height: 1;
  color: #4c54d2;
  margin: 40px 0 28px;
}

.c13 {
  display: inline-block;
  vertical-align: text-top;
}

.c14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background: #fba;
  margin-bottom: 10px;
  border-radius: 6px;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: start;
  align-content: flex-start;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  overflow: hidden;
}

.c15 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-flex-shrink: 1;
  -ms-flex-negative: 1;
  flex-shrink: 1;
  padding: 12px 21px 10px;
}

.c19 {
  background: rgba(251,84,43,0.9);
  -webkit-flex-shrink: 1;
  -ms-flex-negative: 1;
  flex-shrink: 1;
  -webkit-flex-basis: 86px;
  -ms-flex-preferred-size: 86px;
  flex-basis: 86px;
  font-family: Muli;
  font-size: 12px;
  font-weight: bold;
  line-height: 52px;
  text-transform: uppercase;
  text-align: center;
  color: #fff;
  padding: 0 10px;
}

.c19:hover {
  background: rgba(251,84,43,1);
}

.c20 {
  font-size: 12px;
  color: #686978;
  margin-top: 26px;
}

.c21 {
  vertical-align: text-bottom;
  margin-right: 11px;
}

.c6 {
  font-family: Poppins;
  font-weight: 300;
  line-height: 1.4;
  color: #686978;
  font-size: 14px;
  display: inline-block;
}

.c16 {
  font-family: Poppins;
  font-weight: 300;
  line-height: 1.4;
  color: #fff;
  font-size: 22px;
  display: inline-block;
}

.c7 {
  color: #C12D7C;
  font-weight: 600;
}

.c9 {
  color: #9752CB;
  font-weight: 600;
}

.c10 {
  color: #4C54D2;
  font-weight: 600;
}

.c17 {
  color: #fff;
  font-weight: 600;
}

.c8 {
  color: #9E9FAB;
  font-size: 10px;
  font-family: Muli;
  line-height: 1.29;
  display: inline-block;
  margin-left: 5px;
}

.c18 {
  color: rgba(255,255,255,0.65);
  font-size: 14px;
  font-family: Muli;
  line-height: 1.29;
  display: inline-block;
  margin-left: 5px;
}

.c3 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-bottom: 1px solid #d0d6dc;
  border-top: none;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: start;
  align-content: flex-start;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  margin-bottom: 8px;
  font-family: Poppins;
}

.c11 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-bottom: none;
  border-top: none;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: start;
  align-content: flex-start;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  margin-bottom: 8px;
  font-family: Poppins;
}

.c4 {
  font-size: 14px;
  line-height: 2.79;
  color: #4b4c5c;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-flex-shrink: 1;
  -ms-flex-negative: 1;
  flex-shrink: 1;
  -webkit-flex-basis: 60%;
  -ms-flex-preferred-size: 60%;
  flex-basis: 60%;
}

.c5 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-flex-shrink: 1;
  -ms-flex-negative: 1;
  flex-shrink: 1;
  -webkit-flex-basis: 40%;
  -ms-flex-preferred-size: 40%;
  flex-basis: 40%;
  text-align: right;
}

<div
  className="c0"
  id="empty"
>
  <div
    className="c1"
  >
    Rewards Summary
  </div>
  <div
    className="c2"
  >
    May 2018
  </div>
  <div
    className=""
  >
    <div
      className="c3"
      id={undefined}
    >
      <div
        className="c4"
      >
        Token Grant
      </div>
      <div
        className="c5"
      >
        <span
          id={undefined}
        >
          <span
            className="c6"
          >
            <span
              className="c7"
            >
              
              10
            </span>
             
            <span>
              tokens
            </span>
          </span>
          <span
            className="c8"
          >
            ~ 
            0.25
             
            USD
          </span>
        </span>
      </div>
    </div>
    <div
      className="c3"
      id={undefined}
    >
      <div
        className="c4"
      >
        Earnings from Brave Ads
      </div>
      <div
        className="c5"
      >
        <span
          id={undefined}
        >
          <span
            className="c6"
          >
            <span
              className="c7"
            >
              
              10
            </span>
             
            <span>
              tokens
            </span>
          </span>
          <span
            className="c8"
          >
            ~ 
            0.25
             
            USD
          </span>
        </span>
      </div>
    </div>
    <div
      className="c3"
      id={undefined}
    >
      <div
        className="c4"
      >
        Brave Contribute
      </div>
      <div
        className="c5"
      >
        <span
          id={undefined}
        >
          <span
            className="c6"
          >
            <span
              className="c9"
            >
              -
              10
            </span>
             
            <span>
              tokens
            </span>
          </span>
          <span
            className="c8"
          >
            ~ 
            0.25
             
            USD
          </span>
        </span>
      </div>
    </div>
    <div
      className="c3"
      id={undefined}
    >
      <div
        className="c4"
      >
        Recurring Donations
      </div>
      <div
        className="c5"
      >
        <span
          id={undefined}
        >
          <span
            className="c6"
          >
            <span
              className="c10"
            >
              -
              2
            </span>
             
            <span>
              tokens
            </span>
          </span>
          <span
            className="c8"
          >
            ~ 
            0.25
             
            USD
          </span>
        </span>
      </div>
    </div>
    <div
      className="c11"
      id={undefined}
    >
      <div
        className="c4"
      >
        One-time Donations/Tips
      </div>
      <div
        className="c5"
      >
        <span
          id={undefined}
        >
          <span
            className="c6"
          >
            <span
              className="c10"
            >
              -
              19
            </span>
             
            <span>
              tokens
            </span>
          </span>
          <span
            className="c8"
          >
            ~ 
            5.25
             
            USD
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    className="c12"
  >
    <span
      className="c13"
    >
      <svg
        height="25"
        width="36"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          opacity="0.39"
        >
          <path
            d="M20.997 11c-.036-.57-.747-1.016-2.173-1.356a.712.712 0 0 0 .076-.311V7h-.005c-.127-1.943-8.486-2-9.444-2-.96 0-9.32.057-9.446 2H0v2.333c.002.633.86 1.13 2.136 1.499a.716.716 0 0 0-.03.167H2.1v2.33c-1.376.338-2.062.777-2.097 1.337H0v2.335c.002.44.419.813 1.103 1.126a.726.726 0 0 0-.049.207H1.05v2.334c.004 1.108 2.625 1.797 5.538 2.117.02.005.04.013.063.013.007 0 .012-.003.018-.005.563.06 1.133.108 1.697.142.012.002.023.007.035.007.007 0 .012-.004.018-.004.717.04 1.422.062 2.082.062.66 0 1.365-.021 2.082-.062.006 0 .011.004.018.004.012 0 .023-.007.035-.007a36.754 36.754 0 0 0 1.697-.142c.006 0 .011.004.018.004.023 0 .043-.009.063-.012 2.913-.322 5.534-1.01 5.538-2.119v-2.332h-.007c-.024-.409-.39-.75-1.113-1.035a.704.704 0 0 0 .069-.3v-2.18c1.256-.367 2.098-.859 2.1-1.487v-2.333h-.003zm-2.835 6.108l-.119.1c-.184.154-.49.309-.893.458v-1.588c.417-.139.77-.292 1.05-.46V17c0 .034-.013.07-.038.11zm-7.828 1.548l-.153.003c-.125.003-.254.003-.38.005V17l.289-.005.055-.002c.351-.004.705-.016 1.056-.03v1.663c-.075.005-.15.008-.226.012-.212.007-.424.014-.64.019zm-1.528.005c-.16-.004-.32-.007-.478-.012l-.06-.001c-.19-.007-.38-.013-.567-.023v-1.663c.35.016.702.026 1.054.032h.057l.288.005v1.665c-.068 0-.136 0-.203-.002l-.09-.001zm-6.323-.76l-.032-.01v-1.61c.326.085.678.161 1.049.231v1.626a16.89 16.89 0 0 1-.887-.203l-.13-.034zm7.378 2.76l.289.005v1.665c-.484-.003-.95-.016-1.4-.037v-1.665c.35.016.702.026 1.056.032h.055zm1.333 0a46.4 46.4 0 0 0 1.056-.032v1.663c-.45.021-.916.036-1.4.04v-1.667c.096 0 .192-.002.29-.004h.054zM14 20.51v1.655c-.337.034-.687.064-1.05.088V20.59c.354-.022.705-.05 1.05-.08zm.7-.072c.361-.042.712-.09 1.05-.141v1.637c-.33.055-.68.106-1.05.153v-1.649zm1.75-.26c.371-.07.724-.147 1.05-.23v1.61c-.311.086-.663.17-1.05.247v-1.627zm-11.9 0v1.627a15.283 15.283 0 0 1-1.05-.247v-1.61c.326.083.679.162 1.05.23zm.7.119c.338.053.689.1 1.05.141v1.65a27.84 27.84 0 0 1-1.05-.154v-1.637zM7 20.51c.345.03.696.058 1.05.081v1.662A30.567 30.567 0 0 1 7 22.165V20.51zm-.868-1.994a7.127 7.127 0 0 1-.106-.01l-.075-.008v-1.655c.343.032.694.058 1.049.081v1.662c-.294-.021-.585-.044-.868-.07zM5.25 16.77v1.649c-.37-.047-.72-.098-1.05-.153v-1.638c.34.054.69.101 1.05.142zm6.884 1.798l-.234.017v-1.662c.355-.023.705-.049 1.05-.081v1.655c-.217.021-.437.042-.664.06-.051.004-.102.009-.152.011zm1.516-.15v-1.648c.36-.04.71-.088 1.05-.142v1.638c-.329.055-.68.107-1.05.153zm1.75-1.908a16.23 16.23 0 0 0 1.05-.23v1.61c-.312.086-.662.17-1.05.247v-1.627zm-4.785-1.522a37.232 37.232 0 0 1-.775-.028c-.013-.002-.027-.002-.04-.002v-1.665a37.284 37.284 0 0 0 1.11.035l.29.003v1.665c-.17-.002-.343-.003-.508-.006-.026-.002-.052-.002-.077-.002zM2.45 10.225v-1.61c.326.083.678.16 1.049.23v1.627c-.043-.01-.087-.018-.13-.026-.186-.039-.367-.08-.542-.124-.131-.03-.258-.065-.377-.097zm6.36-.897c.096.002.192.003.29.003v1.665c-.472-.005-.94-.016-1.4-.037V9.296c.35.014.702.025 1.054.032h.057zm1.334-.002a47.1 47.1 0 0 0 1.056-.03v1.661l-.178.01-.014.001c-.236.01-.475.017-.718.022-.037.001-.075.001-.112.003-.125.002-.253.003-.38.003V9.331c.098 0 .194-.001.29-.003l.056-.002zm7.975.168l-.01-.002-.128.095c-.184.136-.468.275-.832.41V8.41c.417-.137.77-.29 1.05-.457V9.33c0 .05-.028.106-.08.163zm-4.47 1.257V9.103c.36-.04.71-.087 1.05-.14v1.635c-.332.055-.681.107-1.05.153zm1.75-.282V8.845c.372-.07.723-.147 1.05-.23v1.606c-.314.087-.668.17-1.05.248zm-2.45-1.293v1.654c-.338.033-.688.064-1.05.088v-1.66c.355-.023.705-.05 1.05-.082zm-7 1.654V9.176c.344.032.695.06 1.05.082v1.658a33.343 33.343 0 0 1-1.05-.086zm-.7-1.727v1.648c-.07-.009-.142-.015-.21-.025a26.547 26.547 0 0 1-.725-.106c-.04-.006-.077-.013-.114-.02V8.964c.338.053.689.1 1.049.14zm3.763 5.811a24.958 24.958 0 0 1-.798-.067c-.046-.003-.092-.008-.137-.013-.01 0-.018-.001-.029-.003v-1.655c.346.032.696.058 1.051.08v1.663l-.087-.005zm-2.312-.25c-.138-.02-.268-.043-.402-.066v-1.634c.34.052.69.1 1.051.14v1.649a25.94 25.94 0 0 1-.625-.085c-.008-.002-.015-.003-.024-.003zm-1.1-1.819v1.625c-.389-.08-.74-.165-1.052-.251v-1.605c.328.085.679.161 1.051.231zm-1.75-.434v1.583c-.55-.201-.907-.408-1.02-.592l-.025-.038-.007-.032v-1.38c.28.167.635.32 1.051.459zm8.394.916c.352-.006.704-.016 1.054-.033v1.664c-.46.021-.926.033-1.4.037v-1.663l.29-.003.056-.002zm4.522 1.28c-.154.024-.308.05-.467.072l-.278.04c-.09.011-.181.02-.272.032v-1.647c.36-.04.71-.088 1.05-.14V14.6c-.01.001-.021.005-.033.006zm-2.369.284c-.132.01-.266.016-.398.026v-1.66c.355-.023.706-.049 1.05-.081v1.653c-.204.022-.406.04-.616.059l-.036.003zm4-.625c-.175.046-.36.09-.548.132-.087.02-.176.038-.266.057l-.084.016v-1.626c.373-.07.723-.146 1.05-.231v1.612a5.215 5.215 0 0 0-.153.04zm1.898-3.244c-.033.226-.57.528-1.531.81-1.24.364-3.177.697-5.633.802l-.378.014-.318.008a38.381 38.381 0 0 1-1.774 0l-.316-.01c-.128-.002-.253-.007-.38-.012-2.453-.105-4.391-.438-5.63-.803-.954-.28-1.49-.58-1.532-.804.002-.006.002-.011.007-.02.024.006.05.01.074.017l.42.09a25.35 25.35 0 0 0 2.287.348l.008.001h.002c.559.06 1.14.108 1.735.142.005 0 .008.003.013.003l.007-.002a42.582 42.582 0 0 0 2.094.06c.183 0 .37 0 .562-.004 2.543-.038 4.9-.326 6.539-.771l.222-.064.133-.037c.127-.04.248-.08.366-.121.02-.006.039-.015.06-.021a6.862 6.862 0 0 0 .642-.272l.11-.06.122-.07c.023-.013.049-.026.069-.04 1.567.323 1.98.678 2.02.816zM9.451 5.668c5.77 0 8.69.864 8.745 1.36-.043.226-.581.524-1.532.804-1.239.365-3.177.698-5.63.801l-.38.015-.316.008a38.178 38.178 0 0 1-1.774 0l-.318-.008-.378-.015c-2.456-.103-4.394-.436-5.633-.8-.95-.28-1.488-.579-1.53-.805.055-.496 2.974-1.36 8.746-1.36zM.7 9.333v-1.38c.278.167.632.32 1.049.457v1.597C1.07 9.764.7 9.52.7 9.333zm1.695 4.613l.019.014c.478.46 1.47.824 2.705 1.094a11.394 11.394 0 0 0 .364.075c1.133.23 2.489.395 3.963.478h.004c.475.027.958.044 1.454.052a28.294 28.294 0 0 0 1.386-.001c.458-.008.91-.026 1.354-.05h.005l.008-.002c1.48-.083 2.868-.25 4.042-.492a6.635 6.635 0 0 1-1.036.384c-1.307.386-3.396.736-6.048.817l-.237.007-.26.005a40.873 40.873 0 0 1-1.555-.003l-.318-.01c-.128-.003-.253-.008-.378-.013-2.456-.104-4.394-.438-5.633-.803-.962-.283-1.5-.584-1.531-.81.04-.126.393-.44 1.692-.742zM.701 16.999v-1.38c.278.167.632.32 1.049.459v1.588l-.2-.073-.003.003C.997 17.38.7 17.166.7 17zm1.049 3.667v-1.38a5.4 5.4 0 0 0 1.05.459v1.595c-.68-.243-1.05-.487-1.05-.674zm16.45.674v-1.595a5.4 5.4 0 0 0 1.05-.46v1.38c0 .188-.37.432-1.05.675zm1.045-2.98c-.038.224-.576.524-1.532.805-1.237.365-3.175.699-5.63.803-.127.005-.252.01-.38.013l-.316.01c-.29.004-.585.01-.887.01a40.825 40.825 0 0 1-1.201-.02l-.381-.015c-2.456-.102-4.394-.436-5.631-.8-.927-.272-1.462-.562-1.528-.785.104.034.212.068.323.1l.034.01c.094.028.19.054.29.08.038.01.075.021.113.03.057.014.113.029.17.042 1.276.31 2.903.532 4.66.631h.001c.544.03 1.098.05 1.658.056.149.003.298.003.443.003a38.265 38.265 0 0 0 2.097-.058h.006c.6-.036 1.181-.085 1.74-.146h.007c.007 0 .014-.003.02-.003a26.627 26.627 0 0 0 2.166-.325c1.034-.202 1.928-.464 2.543-.788l.096-.054c.039-.02.076-.044.112-.067.035-.02.072-.04.103-.061.692.234.882.444.904.528zm.005-4.354V12.41c.417-.139.77-.292 1.05-.46v1.38c0 .188-.37.432-1.05.675z"
            fill="#686978"
          />
          <ellipse
            cx="9"
            cy="7.5"
            fill="#FFF"
            rx="8"
            ry="1.5"
          />
          <g
            transform="translate(11)"
          >
            <circle
              cx="13"
              cy="12"
              fill="#FFF"
              r="12"
            />
            <path
              d="M13.147.055c-.425 0-.845.033-1.257.09a9.282 9.282 0 0 0-1.248-.09c-3.07 0-5.843 1.525-7.787 3.96-.02.018-.034.039-.05.06a12.674 12.674 0 0 0-1.61 2.685 13.627 13.627 0 0 0-.736 2.093.466.466 0 0 0-.033.132A14.313 14.313 0 0 0 0 12.452a14.07 14.07 0 0 0 1.031 5.316c.388.95.876 1.83 1.45 2.628.013.02.026.042.044.059.58.797 1.246 1.509 1.98 2.114.02.023.044.04.07.059 1.722 1.398 3.814 2.22 6.067 2.22.406 0 .816-.029 1.248-.09.412.06.832.09 1.257.09 5.87 0 10.645-5.56 10.645-12.396 0-6.837-4.775-12.397-10.645-12.397zM8.741 1.172a9.66 9.66 0 0 0-.41.231c-.075.044-.152.088-.226.134-.116.073-.228.15-.342.227H6.96a8.676 8.676 0 0 1 1.912-.666c-.045.023-.088.05-.13.074zM4.533 19.72H3.037a12.06 12.06 0 0 1-.98-1.71h1.581c.065.15.133.3.204.449a11.679 11.679 0 0 0 .361.697c.034.06.067.124.101.186.074.127.151.252.229.378zM1.865 7.323h1.598c-.074.186-.143.376-.208.566l-.055.158c-.06.184-.117.37-.17.558H1.403c.132-.44.287-.869.463-1.282zm-1.03 5.129c0-.288.015-.572.033-.854h1.663v.019c-.016.277-.026.555-.026.835 0 .144.004.285.008.428H.852c-.006-.143-.016-.284-.016-.428zm1.727 1.284c.014.179.035.357.059.536a13.748 13.748 0 0 0 .114.745H1.088c-.081-.42-.144-.847-.187-1.281h1.66zM2.82 9.46c-.012.05-.02.1-.03.152l-.036.167c-.05.27-.095.538-.13.814l-.004.04c-.006.035-.008.073-.012.108H.952c.055-.434.124-.864.22-1.281H2.82zm.1 6.412l.036.157a13.508 13.508 0 0 0 .245.826c.016.052.034.105.053.159.016.045.03.094.047.14H1.694c-.16-.416-.294-.844-.41-1.282h1.637zm1.383-10.31c-.034.061-.069.124-.101.186a11.997 11.997 0 0 0-.371.718H2.266c.316-.61.68-1.181 1.085-1.71h1.461l-.153.226c-.122.188-.24.382-.355.58zm1.22-2.943h1.153a9.717 9.717 0 0 0-.308.282c-.086.081-.165.167-.249.25-.081.086-.163.172-.244.26-.078.083-.153.169-.229.254-.067.077-.13.157-.195.236H4.074c.448-.478.932-.908 1.45-1.282zM3.682 20.573h1.433a10.559 10.559 0 0 0 .76.919c.081.088.163.175.246.26.033.034.065.072.098.105H4.965a10.426 10.426 0 0 1-1.282-1.284zm5.189 3.232a8.732 8.732 0 0 1-2.7-1.094H7.18c.053.044.108.084.163.123.086.067.171.134.26.196.164.117.331.23.502.337l.225.135c.136.08.272.157.41.23.044.023.087.05.132.073zm4.275.19c-.332 0-.658-.02-.982-.059l-.16-.023-.068-.01a9.503 9.503 0 0 1-.43-.073c-.013-.002-.022-.007-.035-.008a7.283 7.283 0 0 1-.412-.094c-.02-.005-.04-.01-.06-.017-.127-.033-.253-.067-.378-.107a13.8 13.8 0 0 1-.43-.146 8.954 8.954 0 0 1-.252-.102c-.058-.023-.117-.044-.171-.069-2.247-.973-4.107-2.885-5.24-5.329L4.5 17.89a9.237 9.237 0 0 1-.175-.403c-.025-.054-.045-.108-.066-.163-.042-.106-.085-.212-.124-.32l-.073-.21c-.033-.095-.065-.19-.096-.286l-.073-.237c-.027-.09-.054-.182-.078-.272-.024-.083-.044-.169-.067-.253-.022-.09-.045-.18-.065-.269a11.66 11.66 0 0 1-.057-.26c-.02-.093-.037-.182-.055-.274a8.21 8.21 0 0 1-.047-.265c-.016-.094-.029-.19-.043-.284-.012-.088-.027-.173-.036-.26-.013-.103-.023-.205-.033-.31-.009-.081-.019-.16-.025-.242-.01-.119-.016-.238-.024-.359l-.012-.202c-.007-.188-.013-.376-.013-.568 0-.19.006-.38.013-.568l.012-.202c.008-.12.014-.24.024-.36.006-.08.016-.16.025-.242.01-.102.02-.206.033-.308.01-.088.024-.174.036-.262.014-.093.027-.19.043-.284.014-.087.03-.175.047-.262l.055-.276c.018-.088.037-.173.057-.259.02-.092.043-.181.065-.271.023-.084.043-.167.067-.253.024-.09.051-.181.078-.27l.073-.24c.03-.095.063-.189.096-.285l.073-.209a13.625 13.625 0 0 1 .365-.887c.01-.023.018-.046.029-.069 1.133-2.444 2.993-4.356 5.24-5.329.054-.025.113-.046.17-.069.084-.035.168-.069.253-.102.104-.038.21-.073.316-.109A5.918 5.918 0 0 1 11 1.192a6.846 6.846 0 0 1 .473-.11c.013-.002.022-.007.035-.009.142-.027.285-.051.43-.072.022-.005.045-.007.067-.011l.16-.023c.325-.037.651-.056.983-.056 5.41 0 9.81 5.176 9.81 11.54 0 6.365-4.4 11.544-9.81 11.544z"
              fill="#686978"
            />
            <path
              d="M5.864 12.283c0-.275.01-.55.03-.82.017-.239-.149-.45-.372-.47-.224-.016-.418.16-.436.401-.022.292-.033.593-.033.89v.01c0 .24.181.43.405.43.224 0 .406-.2.406-.44zM20.502 10.224a.489.489 0 0 0 .479.336.612.612 0 0 0 .12-.013c.262-.061.423-.301.358-.54-.084-.297-.18-.591-.29-.878-.087-.232-.368-.357-.626-.277-.257.08-.394.334-.305.564.098.264.187.535.264.808zM18.693 18.491a6.804 6.804 0 0 1-.477.525.371.371 0 0 0 .001.53.379.379 0 0 0 .268.107.378.378 0 0 0 .27-.11c.183-.182.361-.376.528-.578a.37.37 0 0 0-.055-.526.385.385 0 0 0-.535.052zM6.024 14.793c-.064-.238-.335-.383-.597-.322-.265.058-.427.299-.359.537.082.297.178.591.286.88.07.184.263.301.468.301a.586.586 0 0 0 .16-.024c.257-.08.395-.332.308-.563a8.573 8.573 0 0 1-.266-.809zM17.478 4.131a8.013 8.013 0 0 0-.82-.452.461.461 0 0 0-.61.19.427.427 0 0 0 .2.584c.252.121.498.26.734.405a.464.464 0 0 0 .628-.125.424.424 0 0 0-.132-.602zM16.198 20.57c-.245.132-.5.25-.755.352a.472.472 0 0 0-.257.598.443.443 0 0 0 .415.298.42.42 0 0 0 .155-.029 7.8 7.8 0 0 0 .846-.394.476.476 0 0 0 .192-.625.433.433 0 0 0-.596-.2zM19.722 6.777c-.155-.279-.321-.55-.494-.804-.14-.206-.389-.236-.557-.067a.56.56 0 0 0-.054.682c.156.23.307.476.447.728.077.14.2.213.329.213a.342.342 0 0 0 .222-.083c.182-.149.228-.45.107-.67zM21.067 11.859c-.224 0-.406.21-.406.452 0 .27-.01.542-.028.808-.018.24.149.453.373.472h.032c.21 0 .388-.174.404-.405.022-.29.032-.584.032-.875v-.028a.41.41 0 0 0-.407-.424zM20.735 15.346a.47.47 0 0 0-.589.284c-.095.27-.2.538-.318.795a.447.447 0 0 0 .24.593.511.511 0 0 0 .187.037.47.47 0 0 0 .428-.27c.127-.283.243-.576.345-.87a.445.445 0 0 0-.293-.569zM13.939 2.815a7.534 7.534 0 0 0-.873-.05h-.017a.412.412 0 0 0-.417.41c0 .226.2.41.434.41.258 0 .517.015.775.042.017.002.033.004.05.004a.421.421 0 0 0 .422-.362.414.414 0 0 0-.374-.454zM7.041 17.7c-.122-.221-.37-.278-.55-.13-.182.149-.23.448-.106.67.152.278.317.55.49.803.079.116.194.177.308.177.089 0 .178-.035.252-.108a.56.56 0 0 0 .054-.681 9.476 9.476 0 0 1-.448-.73zM6.459 8a.471.471 0 0 0-.614.23 10.89 10.89 0 0 0-.347.869.447.447 0 0 0 .29.57.466.466 0 0 0 .59-.281c.094-.268.201-.536.319-.797a.443.443 0 0 0-.238-.592zM13.036 21.434c-.26 0-.522-.016-.777-.046a.42.42 0 0 0-.467.357.414.414 0 0 0 .366.456 7.6 7.6 0 0 0 .878.05h.037c.231 0 .4-.182.4-.407 0-.228-.206-.41-.437-.41zM7.771 5.04c-.181.181-.36.377-.528.577a.374.374 0 0 0 .295.613c.109 0 .219-.048.295-.137.151-.185.313-.36.478-.525a.371.371 0 0 0-.002-.53.386.386 0 0 0-.538.002zM10.771 3.228a7.961 7.961 0 0 0-.845.392.475.475 0 0 0-.194.623.44.44 0 0 0 .394.254.435.435 0 0 0 .201-.05 6.87 6.87 0 0 1 .755-.349.476.476 0 0 0 .26-.598.437.437 0 0 0-.57-.272zM10.28 20.565a7.614 7.614 0 0 1-.736-.41.469.469 0 0 0-.628.128.423.423 0 0 0 .13.6c.266.167.543.32.821.456a.47.47 0 0 0 .206.046.456.456 0 0 0 .403-.237.424.424 0 0 0-.197-.583z"
              fill="#686978"
            />
          </g>
        </g>
      </svg>
    </span>
     Token Grant
  </div>
  <div
    className="c14"
  >
    <div
      className="c15"
    >
      <span
        id={undefined}
      >
        <span
          className="c16"
        >
          <span
            className="c17"
          >
            
            15
          </span>
           
          <span>
            tokens
          </span>
        </span>
        <span
          className="c18"
        >
          ~ 
          0.75
           
          USD
        </span>
      </span>
    </div>
    <div
      className="c19"
      onClick={undefined}
    >
      Claim
    </div>
  </div>
  <div
    className="c14"
  >
    <div
      className="c15"
    >
      <span
        id={undefined}
      >
        <span
          className="c16"
        >
          <span
            className="c17"
          >
            
            10
          </span>
           
          <span>
            tokens
          </span>
        </span>
        <span
          className="c18"
        >
          ~ 
          0.5
           
          USD
        </span>
      </span>
    </div>
    <div
      className="c19"
      onClick={undefined}
    >
      Claim
    </div>
  </div>
  <div
    className="c20"
    onClick={[Function]}
  >
    <span
      className="c21"
    >
      <svg
        height="16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M14.75 14.375a.375.375 0 0 1-.375.375h-8.75a.375.375 0 0 1-.375-.375V13h5.125c.896 0 1.625-.729 1.625-1.625V4.494c0-.409-.153-.801-.434-1.104l-.129-.14h2.938c.207 0 .375.168.375.375v10.75zm-13.5-3v-9.75c0-.207.168-.375.375-.375h4.518v2.822c0 .897.729 1.625 1.625 1.625h2.982v5.678a.375.375 0 0 1-.375.375h-8.75a.375.375 0 0 1-.375-.375zm8.298-8.323a.209.209 0 0 1 .02.022l1.08 1.164a.364.364 0 0 1 .083.209H7.768a.376.376 0 0 1-.375-.375V1.25h.32c.104 0 .204.044.276.12l1.559 1.682zM10.277 2L8.905.521A1.627 1.627 0 0 0 7.713 0H1.625C.729 0 0 .729 0 1.625v9.75C0 12.271.729 13 1.625 13H4v1.375C4 15.271 4.729 16 5.625 16h8.75c.896 0 1.625-.729 1.625-1.625V3.625C16 2.729 15.271 2 14.375 2h-4.098z"
          fill="#A1A8F2"
          fillRule="evenodd"
        />
      </svg>
    </span>
     Wallet Activity/ Monthly Statement
  </div>
</div>
`;
